src/erlang_types/ty_variable.erl

```diff
@@ -9,6 +9,8 @@
 
 -export([new/1, smallest/3, normalize/6, transform/2, get_new_id/0]).
 
+-export_type([var/0]).
```

Comment:
Where is this type used?

Reply:
In src/ast_lib.erl. (I did not introduce the use there, but I had to fix it, I guess, for Erlang 26)